coding-intro/coding-intro-core/core-concepts/functions-are-programs-too.md

```diff
@@ -1,25 +1,45 @@
 ---
 author: kapnobatai136
 
-type: normal
+tags:
+  - discussion
 
-category: must-know
+type: normal
 
+category: discussion
```

Comment:
setting the `category` to `discussion` means the insight will now have the "Give it a try" button which (imo) doesn't make sense here because there is nothing to try

Reply:
Actually, this is supposed to be a discussion insight. I think the plan is to use give it a try right now and then change the label of the button. In any case, the goal is to engage people with an open-ended question and have a button that takes them to the comment section. @nem035 might be able to shed some light on the matter.